.travis.yml

```diff
@@ -1,16 +1,16 @@
 # A Travis CI configuration file.
 
 sudo: required
-
+group: deprecated-2017Q3
 language: scala
-scala:
-   - 2.11.6
+services:
+- docker
+
 env:
   global:
     - secure: "mBHstQIJFTt7klYxMR+GLQ97YA2jDgOVd4vBOCj6clugG6ujKvV0716r3U//4/qI0D7tu0EfSnGTs3xJkSx4Ctfz2xGfAA2leEmZJSoOxcvlayWu41+QLvH4ZCbBBchifvpHR1z2ilZF0owtwo3wlDL51zn+rxx/zmA6daesW7ggG15+uron+d1esbT9nEQVZfIVGOyK6vgcuF4kMPR181p07khG7jbRnH6F3hlox2el3t0ftRZJcADWqlSgNfv1EwCQBbhi/POTNeYbapLxwWhPbgNDHzAYvJrCTWj1M9/6wewp6q1rlDwsukRQA9zVbvjXqf+JCQYoWTsoamj3LARmsUKWLsU9+wOUxwiyZwZNXYGTke7eVvoQv3D+8Zsfv+LGNoYwCaNz5+IX5BdTDi4gIPNyulYanSh7UPG05tHXCXHXVEBPXubxEmXjNvVS9BC2YozSW6GMEyaRSh5lS2Ae9VuLFUSgh+Eernswvlt3KLb7dIhf1wBa/ekecMwsOx+6rMokPl4/LrcbxMbrd/lYYMD98BSgbxBBSyFWJOYfR6toFOGourfV6CP5mtbi/lxaM5LTskj9+VNGVqV0mSZwhpVrqU1Cf/DtrI1B0y7xJRiaCD7opnQ9hKBcXTm0F0zW8TGAv+h5UmDMItt0ZKuMYPpOM4eiR679fOsvmFA="
     - secure: "cZijVhCVtuBSAH/EpjIxfXSjZGjRULM5UYyvKkDZx/FqMd47K0+xzDWIfizowvFa4bGjU2JgjklmlERdF/P3Xp34UDciLvt7q274G8Rqnlq52KMM56iL/LbkeISeaRgbN7lWq+XZnkRDojAIrM67HUn9lsYs4SXAY8HCBXPnzhLN8dRE7BbEGKjQJceUQMxKNfiCVJ7r29qFXbdtjMQ8ea6lHJkczUaLKEFWYGpqIGXJMYAhDGd54OuwqrjOQR1AtNGjyEy1yJRoexKemfrvYl2kUuH6qtJwQv4dQMpSOkVtHOISoiIyHjsxYnCGgKnrLRz+R2F9I1PFxZaVlJEMrG3bc7CDZbXowcnmNEy5FxdAoxgxaTWZwO8mVRnaY/u5hEOkz23Hm/Kmkq5d17XThSrwQh9GG3DZMGabINwo05yAKjSmFV1wbWZmIYzU2ls1H2P+T833TkYqV5PQfOXYFfpV7jrBT0fydgIMIEmPp20Bejf+2KvDcn8ITwR8u25g7TIkmp0CUgv5blWowtGLBH08Ps/yM4YKXX20nNuuAWVYN3OzFAHmtHHAAGaEj0z5xtpKjwSKInFQtuAa3uTG5qMb2/xBxRKhsJmXYVIYRDsGg1Kmk/G97vJSM0WZykUt3EyvfQe4xq9UtHSO5jtnVl4XxXp6gQOTWSuZyUVpIHI="
-services:
-  - docker
+
```

Comment:
Shouldn't need the extra new line here.

Reply:
submitted commit, to remove the extra line